### PR TITLE
Fill in "Author Selects Segment" bullet point of perform_code_review.md

### PR DESCRIPTION
**Pull request by bielsnohr**
Very rough first pass at filling in the first step of the "Perform Code Review" recipe.

### CODE REVIEW

File: site/content/recipes/perform_code_review.md [hunk 1]
```diff
@@ -2,15 +2,36 @@
...
+- Your choice should address the objectives set at the [Meet and Agree on
+  Objectives step](../meet_and_agree_on_objectives). For example, if you want to
+  get feedback on the style of your code, picking _recent additions_ to the code
+  will be more reflective of your own style rather than portions that you have
+  _amended_, since they might have been originally written by someone else.
+  Alternatively, if you want a general overview of the readability of your code,
+  pick a snippet that shares characteristics with many other areas of the code
+  base so that the feedback on this particular snippet can be extrapolated more
+  broadly. This applies in general: the more representative of the codebase the
+  snippet is, the more widely relevant the feedback.
```
Comment (tlestang) [inline site/content/recipes/perform_code_review.md:21]: ^ I think this paragraph reads really well!

File: site/content/recipes/perform_code_review.md [hunk 2]
```diff
@@ -2,15 +2,36 @@
...
-    - We suggest picking a piece of code that is 50-100 lines.
-    - If the code you plan to review is larger than that, agree on a strategy to sequentially go through smaller segments, or prioritise the most important segment
-    - Choice is helped by objectvies from MAO stage.
-      - TODO add examples: e.g. if coding style is reviewed, pick recently added class or method. The more representative of the codebase the snippet is, the more widely relevant the feedback.
-      - Author should be familiar with the details of the chosen snippet. The main idea here is to come prepared.
-
-It's a good idea to share the code to review with the reviewer ahead
+The first step initiating the actual code review phase is for the code author to
+select a segment of their code to be reviewed. This sounds obvious and perhaps
+even easy, but it can be quite a tricky task. The success and impact of the code
```
Comment (tlestang) [inline site/content/recipes/perform_code_review.md:7]: I understand you want to motivate readers by saying choosing the code under review is harder than it sounds, btu as a read er I'm left with the questions: "what is hard about it?". Maybe, _maybe_ skip that sentence alltogether and jump straight to the discussion of how important this choice is for the code review (next sentence)?

Comment (NickleDave) [reply site/content/recipes/perform_code_review.md:7]: +1 agree with what @tlestang says here. I also like the concrete examples in the second chunk below, but I think you could communicate roughly the same thing by skipping the sentence about how "this sounds obvious" and instead just saying:

> The first step initiating the actual code review phase is for the code author to select a segment of their code to be reviewed. Your choice should address the objectives set at the [Meet and Agree on Objectives step](../meet_and_agree_on_objectives). ...

File: site/content/recipes/perform_code_review.md [hunk 3]
```diff
@@ -2,15 +2,36 @@
...
+  Objectives step](../meet_and_agree_on_objectives). For example, if you want to
+  get feedback on the style of your code, picking _recent additions_ to the code
+  will be more reflective of your own style rather than portions that you have
+  _amended_, since they might have been originally written by someone else.
+  Alternatively, if you want a general overview of the readability of your code,
+  pick a snippet that shares characteristics with many other areas of the code
+  base so that the feedback on this particular snippet can be extrapolated more
+  broadly. This applies in general: the more representative of the codebase the
+  snippet is, the more widely relevant the feedback.
+- We recommend selecting a snippet of code that is between **50 and 100 lines long**
```
Comment (NickleDave) [inline site/content/recipes/perform_code_review.md:22]: Perfect. And thank you for doing all the research on this

**Review by NickleDave — APPROVED** [review]
If you agree with @tlestang about that one sentence then you might consider removing or revising but other than that I'd say let's go ahead and merge in

File: site/content/recipes/perform_code_review.md (hunk 2 above)
Comment (bielsnohr) [reply site/content/recipes/perform_code_review.md]: Applied patch in separate comment to address this: https://github.com/ResearchCodeReviewCommunity/dev-review/pull/55/files/3286ada701535fdd396ff9ae187a2d0937cda0e3#r833547419

**Pull request merged**